src/styles/home.css

```diff
@@ -0,0 +1,64 @@
+.like-icon {
+  width: 18px;
+  cursor: pointer;
+}
+
+.like-icon.active {
+  transform: scale(1.05);
+}
+
+.show-items {
+  padding: 16px;
+}
+
+.show-items li {
+  list-style-type: none;
+}
+
+.show-image {
+  width: 100%;
+}
+
+.title-like {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+}
+
+.like-text {
+  text-align: right;
+  margin-block: 10px;
+  font-size: 14px;
+}
+
+.comment-btn {
+  background: var(--primary-txt);
+  color: white;
+  padding: 10px;
+  border: none;
+  border-radius: 4px;
+  width: 100%;
+  margin-bottom: 31px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  cursor: pointer;
+}
+
+@media (min-width: 768px) {
+  .show-items {
+    padding: 25px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 3%;
```

Comment:
- [ ] It is better  to remove the `gap:3%` because this makes your `ul` list container overflow out of the main look at the screenshot below.
![image](https://user-images.githubusercontent.com/30173722/188153888-dbf1e970-bed7-42e9-bfc3-6015178c4daa.png)

**I would recommend you to replaces the gap with**
`  column-gap: 24px;
    row-gap: 24px;`

Here is a new change 
![image](https://user-images.githubusercontent.com/30173722/188154369-c29fd3e3-616e-49e8-a6d8-2dbbb2b01f5f.png)